w3-account.md

```diff
@@ -0,0 +1,224 @@
+
+
+## Verification
+
+Service MUST perform out of bound user verification e.g. send a link to the mailbox of the account by clicking which user would authorize agent to represent the account.
+
+On succesful verification service MUST delegate corresponding `ucan/sign` capability to the [`did:key`][] it was requested [`with`][issue `with`].
+
+Delegation represents authorization to issue [UCAN][]s with [`did:mailto`] account prinicipal which MAY be signed with [`did:key`] of the agent.
+
+
+```ts
+{
+  iss: "did:dns:web3.storage",
+  aud: "did:mailto:alice@web.mail",
+  att: [{
+    with: "did:dns:web3.storage",
+    can: "ucan/sign",
+    nb: { as: "did:key:zAgent" }
+  }],
+  exp: null
+  sig: "..."
+}
+```
+
+### sign `with`
+
+Authorization context, implying that this authorization MUST be considered valid by this recipient (`aud` matches this `with`).
+
+Other recipients MAY also recognize authorizations issued by trusted principals.
+
+
+### sign `aud`
+
+Audience of the [UCAN][] MUST be [`did:mailto`][] identifier of the account principal. This ensures that [principal alignment] requirement can be met when authorization is used as proof by an account.
+
+
+### sign `as`
+
+MUST be a [`did:key`][] of the principal which MAY sign [UCAN][]s issued by an account principal in [`aud`](#sign-aud).
+
+
+## Utilization
+
+Authorized agents MAY issue UCANs using account [`did:mailto`][] identifier, sign it with agent private key and add authorization to `prf` as proofs that it was authorized by recepient to do so.
+
+```ts!
+{
+  // UCAN issued by the account DID
+  iss: "did:mailto:alice@web.mail",
+  aud: "did:dns:web3.storage",
+  // list stored data in did:key:zAlice space
+  att: [{
+    with: "did:key:zAlice",
+    can: "store/list"
+  }],
+  prf: [
+    // Proof that did:key:zAgent MAY sign UCANs issued
+    // by did:mailto:alice@web.mail
+    {
+      iss: "did:dns:web3.storage",
+      aud: "did:mailto:alice@web.mail",
+        att: [{
+          with: "did:dns:web3.storage",
+          can: "ucan/sign",
+          nb: { as: "did:key:zAgent" }
+        }],
+        exp: null
+        sig: "..."
+    },
+    // Proof that did:mailto:alice@web.mail has capability
+    // to list stored data in did:key:zAlice space
+    {
+      iss: "did:key:zAlice",
+      aud: "did:mailto:alice@web.mail",
+      att: [{
+        with: "did:key:zAlice",
+        can: "*"
+      }],
+      exp: null,
+      sig: "..."
+    }
+  ]
+}
+```
+
+
+# Limitations
+
+Using delegation from specific authority as an authorization proof limits it to the contexts where it is trusted. It is reasonable complomise when issued UCAN receipent is the same authority, but problematic in wider contexts.
+
+Long term we would like to upgrade this specification to [UCAN mailto][] so that authorazation may be verifiable and not require trusting third-party that performed [verification][].
```

Comment:
```suggestion
Long term we would like to upgrade this specification to [UCAN mailto][] so that authorization may be verifiable and not require trusting third-party that performed [verification][].
```